app/util/data_preparation/api/abstract_clients.py

```diff
@@ -29,6 +29,12 @@ class RestClient(Client):
         "Accept": "application/json",
         "Content-Type": "application/json"
     }
+    LOGIN_POST_HEADERS = {
+            'Upgrade-Insecure-Requests': '1',
```

Comment:
Need to check if everything still works with HTTPS jira/conf/bbs.

Reply:
This parameter was removed from headers. It works perfectly without this header.